Rakefile

```diff
@@ -3,11 +3,24 @@
 require 'rubygems'
 require 'rake'
 require 'rspec/core/rake_task'
-require 'coveralls/rake/task'
 
 desc 'Run RSpec'
 RSpec::Core::RakeTask.new do |t|
   t.verbose = false
 end
 task default: :spec
-Coveralls::RakeTask.new
+
+# must be first!
+require 'simplecov'
+require 'simplecov-lcov'
+SimpleCov::Formatter::LcovFormatter.config do |c|
+  c.report_with_single_file = true
+  c.single_report_path = 'coverage/lcov.info'
+end
+SimpleCov.formatters = SimpleCov::Formatter::MultiFormatter.new(
+  [
+    SimpleCov::Formatter::HTMLFormatter,
+    SimpleCov::Formatter::LcovFormatter,
+  ]
+)
+SimpleCov.start
```

Comment:
This is all working, but something is going wrong with the coveralls.io push.  This is fine for now.